### PR TITLE
React Capstone

### PR DESCRIPTION
**Pull request by malikhaiderkhan**
In this capstone project I completed all the requirements that is required. I fetch the data from the restcountries API and then display it as required.

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,13 +1,27 @@
...
+    -apple-system,
+    BlinkMacSystemFont,
+    'Segoe UI',
+    'Roboto',
+    'Oxygen',
+    'Ubuntu',
+    'Cantarell',
+    'Fira Sans',
+    'Droid Sans',
+    'Helvetica Neue',
```
Comment (Munsa1) [inline src/index.css]: - [ ] As per the given design guidelines, please use the given font family so as to try and match the given design as much as possible.
image

File: src/__test__/pure_functions.test.js [hunk 2]
```diff
@@ -0,0 +1,1012 @@
...
+            common: 'Ecuador',
+            official: 'Republic of Ecuador',
+            nativeName: {
+              spa: {
+                official: 'República del Ecuador',
+                common: 'Ecuador',
+              },
+            },
+          },
+          tld: [
```
Comment (Munsa1) [inline src/__test__/pure_functions.test.js]: - [ ] You have a failing test case here, It looks like you're encountering a timeout issue while running a test suite using Jest. The error message suggests increasing the timeout using jest.setTimeout(newTimeout). Kindly fix this
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @malikhaiderkhan, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by malikhaiderkhan** [comment]
Now I implemented the suggested changes as required by reviewer. I change the font family to gill sans and increase the timeout for the tests to be passed.

**Review by tugsem — APPROVED** [review]
## STATUS: APPROVED ✔️ 
Hi @malikhaiderkhan , 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

Alt Text
Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**